ui/package.json

```diff
@@ -32,11 +32,12 @@
     "format:types": "prettier --write \"types/**/*.{d.ts,ts,json}\"",
     "test:create": "node ./test/cypress/helpers/create-spec.js -c",
     "test:build": "node build/script.build.js js api && node build/script.build.js js transforms",
-    "test:e2e": "yarn test:build && cd ./dev && cross-env NODE_ENV=test start-test \"node ./script.dev.js\" http-get://localhost:9000 \"cypress open --e2e\" && cd ..",
-    "test:e2e:ci": "yarn test:build && cd ./dev && cross-env NODE_ENV=test start-test \"node ./script.dev.js\" http-get://localhost:9000 \"cypress run --e2e\" && cd ..",
-    "test:component": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress open --component && cd ..",
-    "test:component:run": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress run --component && cd ..",
-    "test:component:ci": "yarn test:build && cd ./dev && cross-env NODE_ENV=test cypress run --component --record --parallel && cd .."
+    "test:e2e": "yarn test:build && cross-env NODE_ENV=test start-test \"node ./dev/script.dev.js\" http-get://localhost:9000 \"cypress open --project ./dev --e2e\"",
+    "test:e2e:run": "cross-env NODE_ENV=test start-test \"node ./dev/script.dev.js\" http-get://localhost:9000 \"cypress run --project ./dev --e2e\"",
```

Comment:
`test:e2e` and `test:e2e:run` reference q-app-vite internal commands `dev` and `build`, which can only be executed inside a Quasar project, hence why there was a `cd ./dev` and related `cd ..` in test commands

I guess it works with the root level monorepo due to it being a monorepo and hoisting some dependencies, but those scripts actually break down when executing inside `ui` folder during normal development

`component` scripts work because they don't rely on Quasar own `dev` and `build` commands, but we kept them with the change directory commands to keep consistency

Reply:
As there are not many e2e tests and they are not important now, it's possible that I might have overlooked this. I have adjusted the scripts in ui/dev to ensure cwd is set correctly. `cd ./dev && ... && cd ..` is both annoying, can stay stuck in a directory in case of fail, and is not possible to pass extra arguments to, so I prefer avoiding it.